src/components/Sing/SideBar/TrackItem.vue

```diff
@@ -0,0 +1,381 @@
+
+watch(
+  () => track.value.name,
+  () => {
+    temporaryTrackName.value = track.value.name;
+  },
+);
+
+const temporaryTrackName = ref(track.value.name);
+
+const updateTrackName = () => {
+  if (temporaryTrackName.value === track.value.name) return;
+  if (temporaryTrackName.value === "") {
+    temporaryTrackName.value = track.value.name;
+    return;
+  }
+  setTrackName(temporaryTrackName.value);
+};
+
+const setTrackName = (name: string) => {
+  if (store.state.songUndoableTrackOptions.name) {
+    store.dispatch("COMMAND_SET_TRACK_NAME", { trackId: props.trackId, name });
+  } else {
+    store.dispatch("SET_TRACK_NAME", { trackId: props.trackId, name });
+  }
+};
+
+const setTrackMute = (mute: boolean) => {
+  if (store.state.songUndoableTrackOptions.soloMute) {
+    store.dispatch("COMMAND_SET_TRACK_MUTE", { trackId: props.trackId, mute });
+  } else {
+    store.dispatch("SET_TRACK_MUTE", { trackId: props.trackId, mute });
+  }
+};
+
+const setTrackSolo = (solo: boolean) => {
+  if (store.state.songUndoableTrackOptions.soloMute) {
+    store.dispatch("COMMAND_SET_TRACK_SOLO", { trackId: props.trackId, solo });
+  } else {
+    store.dispatch("SET_TRACK_SOLO", { trackId: props.trackId, solo });
+  }
+};
+
+const selectedTrackId = computed(() => store.state.selectedTrackId);
+
+const trackCharacter = computed<
+  { character: CharacterInfo; style: StyleInfo } | undefined
+>(() => {
+  if (!track.value.singer) return undefined;
+
+  for (const character of store.state.characterInfos[
+    track.value.singer.engineId
+  ]) {
+    for (const style of character.metas.styles) {
+      if (style.styleId === track.value.singer.styleId) {
+        return { character, style };
+      }
+    }
+  }
+  return undefined;
+});
+const selectTrack = () => {
+  store.dispatch("SET_SELECTED_TRACK", { trackId: props.trackId });
+};
+
+const deleteTrack = () => {
+  store.dispatch("COMMAND_DELETE_TRACK", { trackId: props.trackId });
+};
+
+const singerName = computed(() => {
+  const character = trackCharacter.value;
+  if (!character) return "（不明なキャラクター）";
+  return character.character.metas.speakerName;
+});
+</script>
+<style scoped lang="scss">
+@use "@/styles/colors" as colors;
+@use "@/styles/variables" as vars;
+
+.track-detail-container {
+  padding: 0;
+
+  border-bottom: 1px solid colors.$sequencer-sub-divider;
+
+  .dragging & {
+    display: none;
+  }
+}
+
+.track-detail {
+  margin-left: 0.5rem;
+  padding: 0 0.5rem 0.25rem 0.5rem;
+  width: 100%;
+  border-left: 1px solid colors.$sequencer-sub-divider;
+  display: flex;
+  flex-direction: column;
+
+  .pan,
+  .volume {
+    display: grid;
+    align-items: center;
+    gap: 1rem;
+    grid-template-columns: 1.5rem 1fr 1.5rem;
+
+    .l,
+    .r {
+      justify-self: center;
+    }
+  }
+}
+
+.track-item {
+  padding-right: 0.5rem;
+
+  &.selected-track {
+    color: colors.$display;
+  }
+
+  .track-handle-bg {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 4.5rem;
```

Comment:
将来デザイン変わるかもだし、この大きさの意図ちょっと書いておきたいかも。
こんな感じ？
```suggestion
    width: 4.5rem; // 大体アイコンの辺りまで
```